### PR TITLE
Nano: update comment for input spec

### PR DESCRIPTION
**Pull request by rnwang04**
## Description

Update comment for input spec in keras InferenceOptimizer to make it more clear

### How to test?
- [x] Unit test

### CODE REVIEW

File: python/nano/src/bigdl/nano/tf/keras/inference/optimizer.py [hunk 1]
```diff
@@ -136,8 +136,10 @@ def optimize(self, model: Model,
                   If x is a dataset, y will be ignored (since targets will be obtained from x).
         :param validation_data: (optional) An unbatched tf.data.Dataset object for accuracy
                evaluation. This is only needed when users care about the possible accuracy drop.
-        :param input_spec: A (tuple or list of) ``tf.TensorSpec`` defining the
-                           shape/dtype of the input.
+        :param input_spec: (optional) A (tuple or list of) ``tf.TensorSpec``
+                           defining the shape/dtype of the input. This is only required when
+                           you have a custom Keras model(no input/output layer is explicitly
```
Comment (Oscilloscope98) [inline python/nano/src/bigdl/nano/tf/keras/inference/optimizer.py:141]: A space seems to be missing between before the "("

**Review by Oscilloscope98 — APPROVED** [review]
Others LGTM :)

File: python/nano/src/bigdl/nano/tf/keras/inference/optimizer.py (hunk 1 above)
Comment (rnwang04) [reply python/nano/src/bigdl/nano/tf/keras/inference/optimizer.py]: done

**Review by MeouSker77 — APPROVED** [review]
LGTM

**Pull request merged**